core/src/components/item-sliding/item-sliding.tsx

```diff
@@ -116,6 +116,24 @@ export class ItemSliding implements ComponentInterface {
     return Promise.resolve(this.getSlidingRatioSync());
   }
 
+  /**
+   * Open the sliding item.
+   */
+  @Method()
+  async open() {
+    if (this.openAmount !== 0 && this.item !== null) {
```

Comment:
```suggestion
    if (this.openAmount === 0 && this.item !== null) {
```

I think this should be checking if `openAmount` is 0, otherwise the item will never open if closed.

Reply:
Ah yes, I agree, it should be an equals check here.
Making this change now.